config/options_test.go

```diff
@@ -1300,6 +1300,65 @@ func TestOptions_RuntimeFlags(t *testing.T) {
 	}
 }
 
+func TestOptions_GetDataBrokerStorageConnectionString(t *testing.T) {
+	t.Parallel()
+
+	t.Run("validate", func(t *testing.T) {
+		t.Parallel()
+
+		o := NewDefaultOptions()
+		o.Services = "databroker"
+		o.DataBrokerStorageType = "postgres"
+		o.SharedKey = cryptutil.NewBase64Key()
+
+		assert.ErrorContains(t, o.Validate(), "missing databroker storage backend dsn",
+			"should validate DSN")
+
+		o.DataBrokerStorageConnectionString = "DSN"
+		assert.NoError(t, o.Validate(),
+			"should have no error when the dsn is set")
+
+		o.DataBrokerStorageConnectionStringFile = "DSN_FILE"
```

Comment:
Should we also test when `DataBrokerStorageConnectionStringFile` is set and `DataBrokerStorageConnectionString` is not set?